debian/changelog

```diff
@@ -1,3 +1,57 @@
+snapcraft (2.41) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * package: ensure all relevant files are in for sdist (#2060)
+  * errors: enable sending tracebacks to sentry (#1961) (Closes: #1918)
```

Comment:
Probably don't want the Closes in here, do we?

Reply:
I was adding them, but apparently we all need to use `Closes`